test/e2e/util/kubectl.go

```diff
@@ -104,6 +116,43 @@ func KubectlDescribe(kubeconfigPath, podName, namespace string) (string, error)
 	return Kubectl(args)
 }
 
+// KubectlDescribe executes "kubectl describe" given a list of arguments.
+func KubectlCurlPod(kubeconfigPath string) (string, error) {
+	args := []string{
+		"run",
+		"temp",
+		"--image",
+		"curlimages/curl",
+		"--",
+		"tail",
+		"-f",
+		"/dev/null",
+	}
+	return Kubectl(args)
+}
+
+func KubectlDescribeService(kubeconfigPath, serviceName, namespace string) (string, error) {
+	args := []string{
+		"describe",
+		"service",
+		serviceName,
+		fmt.Sprintf("--kubeconfig=%s", kubeconfigPath),
+		fmt.Sprintf("--namespace=%s", namespace),
+	}
+	return Kubectl(args)
+}
+
+func KubectlPortForward(kubeconfigPath, podName, namespace string) error {
```

Comment:
Not necessary. You only need a port-forward if you're trying to reach a pod from outside of the cluster.